README.md

```diff
@@ -142,3 +143,34 @@ Clean up your local development clusters with
 ```shell script
 make destroy-full-local-setup
 ```
+
+
+## Sample demo
+
+### Non-deterministic Round Robin
```

Comment:
If we talk about non-determinism then we have to explain the meaning...

Also it is definitely worth to mention manifest https://github.com/AbsaOSS/ohmyglb/blob/master/deploy/crds/ohmyglb.absa.oss_v1beta1_gslb_cr.yaml that was used to create testing gslb configuration and associated strategy